### PR TITLE
Feature/player 542

### PR DESCRIPTION
**Pull request by yvinokurov**
This PR is an implementation for the ticket https://jira.corp.ooyala.com/browse/PLAYER-542

We have 2 cases in this PR:
- The first case is an UI for small screen resolutions. For this case a new screen was created.
- The second case is an UI for large screen resolutions. For this case a popover was used.

Unit tests added and passed without errors

### CODE REVIEW

File: js/components/base-components/listItem.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    if (this.props.selected) {
+      for (var key in classes){
+        classes[key] = classes[key] + ' select';
+      }
+    }
+
+    return (
+      <div onClick={this.handleSelect} className={classes.listItem}>
+        <span className={classes.icon}> X </span>
+        {/*<Icon skinConfig={this.props.skinConfig} icon="selectedIcon" className={classes.icon} />*/}
```
Comment (IvanSakharovskii) [inline js/components/base-components/listItem.js]: remove commented code

File: js/components/base-components/listItem.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+var Icon = require('../icon');
+var classnames = require('classnames');
+
+var listItem = React.createClass({
+  handleSelect: function () {
```
Comment (IvanSakharovskii) [inline js/components/base-components/listItem.js]: Is this handleAudioSelect? If it is, please rename

File: js/components/close-caption-multi-audio-menu/closeCaptionMultiAudioMenu.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+    return multiAudio;
+  },
+
+  /**
+   * Getting the list of available languages for subtitles
+   * @return {object} list of available languages for subtitles and selected language
+   * @private
+   */
+  _getCloseCaptions: function () {
```
Comment (IvanSakharovskii) [inline js/components/close-caption-multi-audio-menu/closeCaptionMultiAudioMenu.js]: Closed

File: js/components/base-components/listItem.js (hunk 2 above)
Comment (yvinokurov) [reply js/components/base-components/listItem.js]: it is base component

**Pull request merged**